english/eating-walls/light-fire/fire.md

```diff
@@ -0,0 +1,4 @@
+You build a camp-fire, grab a fistful of the wall and put it on a stick.
```

Comment:
I can't seem to find where to go to start the story.  Where do I go to start language.md?
Thank you.
William F. Baur